.github/workflows/triage-on-comment.yml

```diff
@@ -0,0 +1,102 @@
+name: Triage on Comment
+
+on:
+  issue_comment:
+    types: [created]
+
+permissions:
+  issues: write
+  pull-requests: write
+
+env:
+  LABEL_AWAITING_TEAM: "awaiting team"
+  LABEL_AWAITING_CUSTOMER: "awaiting customer"
+  APPLY_TO_PULL_REQUESTS: "false"
+  TEAM_USERNAMES: ""  # Comma-separated list, e.g. "user1,user2,user3"
+  PERMISSION_MODE: "write-access"  # Either "users" or "write-access"
+
+jobs:
+  label-comment:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check user permissions
+        id: check-permissions
+        uses: actions/github-script@v8
+        with:
+          script: |
+            const issueNumber = context.issue.number;
+            const issueType = context.payload.issue.pull_request ? 'PR' : 'Issue';
+            
+            // Skip if issue is a PR and not configured to support PR
+            if (issueType === 'PR' && process.env.APPLY_TO_PULL_REQUESTS !== 'true'){
+              console.log(`Skipping Issue ${issueNumber} is a PR and PRs are not configured.`);
+              core.setOutput('skip', 'true');
+              return;
+            } 
+            
+            // Skip if commenter is a bot
+            if (context.actor.endsWith('[bot]')) {
+              console.log(`Skipping bot user: ${context.actor}`);
+              core.setOutput('skip', 'true');
+              return;
+            }
+            
+            let hasWriteAccess = false;
+            const permissionMode = process.env.PERMISSION_MODE;
+            
+            console.log(`Processing ${issueType} #${issueNumber}, commenter: ${context.actor}, mode: ${permissionMode}`);
+            
+            if (permissionMode === 'users') {
+              const teamUsernames = process.env.TEAM_USERNAMES;
+              const userList = teamUsernames ? teamUsernames.split(',').map(u => u.trim()) : [];
+              hasWriteAccess = !userList.includes(context.actor);
```

Comment:
shouldn't this be the other way around? `userList` is the list of team members right?